golem-template-service/src/config.rs

```diff
@@ -114,7 +99,6 @@ impl Default for TemplateServiceConfig {
             db: DbConfig::default(),
             template_store: TemplateStoreConfig::default(),
             compilation: TemplateCompilationConfig::default(),
-            worker_executor_client_cache: WorkerExecutorClientCacheConfig::default(),
```

Comment:
I am not sure why we had this. it's not used anywhere

Reply:
GRPC client cache for worker comms from template.  Its confusing we are not using this, something to consider

Reply:
The template service is not calling worker executor at all, right?

Reply:
Oh okay, Ok it got not used after we have a separate worker-service that talks to worker-executor.